charts/sysdig-deploy/README.md

```diff
@@ -201,15 +212,15 @@ agent:
 
 ## NodeAnalyzer
 
-For possible configuration values of the node-analyzer, please refer to the node-analyzer subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/README.md). All agent-specific configuration can be prefixed with `nodeAnalyzer.` to apply them to this chart.
+For possible configuration values of the node-analyzer, please refer to the node-analyzer subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/README.md). All specific configuration can be prefixed with `nodeAnalyzer.` to apply them to this chart.
```

Comment:
For configuration values of the `node-analyzer`, see the node-analyzer subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/README.md).  Prefix all the specific configurations with `nodeAnalyzer.` to apply them to the chart.